Crash on launch with RealmMigrationNeededException

#### Describe the bug
Crash on launch. See stack trace at the end.

#### To Reproduce
Steps to reproduce the behavior:
1. Click to launch Application
2. Application Crashes with Fatal Exception on launch

#### Expected behavior
Application launching and working as expected.

#### Additional context
 - App version and store: F-Droid universal debug apk produced as build artifact of build [2288](https://buildkite.com/matrix-dot-org/element-android/builds/2288) and onwards.
 - Homeserver:matrix.org

Add any other context about the problem here.
Stack Trace:
```
FATAL EXCEPTION: main
Process: im.vector.app.debug, PID: 14050
java.lang.RuntimeException: Unable to create application im.vector.app.VectorApplication: io.realm.exceptions.RealmMigrationNeededException: Migration is required due to the following errors:
- Property 'SpaceChildSummaryEntity.suggested' has been added.
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5925)
	at android.app.ActivityThread.access$1100(ActivityThread.java:200)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1656)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6718)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
Caused by: io.realm.exceptions.RealmMigrationNeededException: Migration is required due to the following errors:
- Property 'SpaceChildSummaryEntity.suggested' has been added.
	at io.realm.internal.OsSharedRealm.nativeGetSharedRealm(Native Method)
	at io.realm.internal.OsSharedRealm.<init>(OsSharedRealm.java:173)
	at io.realm.internal.OsSharedRealm.getInstance(OsSharedRealm.java:249)
	at io.realm.BaseRealm.<init>(BaseRealm.java:141)
	at io.realm.BaseRealm.<init>(BaseRealm.java:108)
	at io.realm.Realm.<init>(Realm.java:159)
	at io.realm.Realm.createInstance(Realm.java:495)
	at io.realm.RealmCache.createInstance(RealmCache.java:481)
	at io.realm.RealmCache.doCreateRealmOrGetFromCache(RealmCache.java:448)
	at io.realm.RealmCache.createRealmOrGetFromCache(RealmCache.java:412)
	at io.realm.Realm.getInstance(Realm.java:424)
	at org.matrix.android.sdk.internal.database.SessionRealmConfigurationFactory.create(SessionRealmConfigurationFactory.kt:79)
	at org.matrix.android.sdk.internal.session.SessionModule$Companion.providesRealmConfiguration(SessionModule.kt:189)
	at org.matrix.android.sdk.internal.session.SessionModule.providesRealmConfiguration(Unknown Source:2)
	at org.matrix.android.sdk.internal.session.SessionModule_ProvidesRealmConfigurationFactory.providesRealmConfiguration(SessionModule_ProvidesRealmConfigurationFactory.java:36)
	at org.matrix.android.sdk.internal.session.SessionModule_ProvidesRealmConfigurationFactory.get(SessionModule_ProvidesRealmConfigurationFactory.java:26)
	at org.matrix.android.sdk.internal.session.SessionModule_ProvidesRealmConfigurationFactory.get(SessionModule_ProvidesRealmConfigurationFactory.java:11)
	at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
	at org.matrix.android.sdk.internal.session.DefaultSession_Factory.get(DefaultSession_Factory.java:237)
	at org.matrix.android.sdk.internal.session.DefaultSession_Factory.get(DefaultSession_Factory.java:54)
	at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
	at org.matrix.android.sdk.internal.session.DaggerSessionComponent.session(DaggerSessionComponent.java:2060)
	at org.matrix.android.sdk.internal.SessionManager.getOrCreateSession(SessionManager.kt:42)
	at org.matrix.android.sdk.internal.auth.DefaultAuthenticationService.getLastAuthenticatedSession(DefaultAuthenticationService.kt:75)
	at im.vector.app.VectorApplication.onCreate(VectorApplication.kt:164)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1154)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5920)
```
Please take a look at this @bmarty

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -8,7 +8,7 @@ Improvements 🙌:
  -
 
 Bugfix 🐛:
- -
+ - Fix a problem with database migration on nightly builds (#3335)
 
 Translations 🗣:
  -

--- a/matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/RealmSessionStoreMigration.kt
+++ b/matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/RealmSessionStoreMigration.kt
@@ -44,7 +44,7 @@ import javax.inject.Inject
 class RealmSessionStoreMigration @Inject constructor() : RealmMigration {
 
     companion object {
-        const val SESSION_STORE_SCHEMA_VERSION = 12L
+        const val SESSION_STORE_SCHEMA_VERSION = 13L
     }
 
     override fun migrate(realm: DynamicRealm, oldVersion: Long, newVersion: Long) {
@@ -62,6 +62,7 @@ class RealmSessionStoreMigration @Inject constructor() : RealmMigration {
         if (oldVersion <= 9) migrateTo10(realm)
         if (oldVersion <= 10) migrateTo11(realm)
         if (oldVersion <= 11) migrateTo12(realm)
+        if (oldVersion <= 12) migrateTo13(realm)
     }
 
     private fun migrateTo1(realm: DynamicRealm) {
@@ -271,6 +272,16 @@ class RealmSessionStoreMigration @Inject constructor() : RealmMigration {
                 }
 
         realm.schema.get("SpaceChildSummaryEntity")
+                ?.addField(SpaceChildSummaryEntityFields.SUGGESTED, Boolean::class.java)
+                ?.setNullable(SpaceChildSummaryEntityFields.SUGGESTED, true)
+    }
+
+    private fun migrateTo13(realm: DynamicRealm) {
+        Timber.d("Step 12 -> 13")
+
+        // Fix issue with the nightly build. Eventually play again the migration which has been included in migrateTo12()
+        realm.schema.get("SpaceChildSummaryEntity")
+                ?.takeIf { !it.hasField(SpaceChildSummaryEntityFields.SUGGESTED) }
                 ?.addField(SpaceChildSummaryEntityFields.SUGGESTED, Boolean::class.java)
                 ?.setNullable(SpaceChildSummaryEntityFields.SUGGESTED, true)
     }

